Documentation/Blazorise.Docs/Pages/Blog/2022-08-23_WhatToExpectInNET7/Index.md

```diff
@@ -0,0 +1,107 @@
+
+In our example below, the number will be set to zero 
+
+![Auto-Default Struct](img/blog/2022-08-23/AutoDefaultStruct.png align="left")
+
+#### Generic Attributes
+An attribute with a generic type can be used as part of an attribute within a class as a fully constructed type.
+
+
+![Generic Attributes](GenericAttributeL.png align="left")
+
+### Extended MAUI Support and Blazor Hybrid
+
+At the time of writing, .NET MAUI is in preview 13. The official release of MAUI will ship with .NET 7 but based on .NET 6 with tooling and performance improvements.
+
+> Suggested Reading - [What Is .NET MAUI](/blog/a-beginners-guide-to-maui)
+
+Blazor Hybrid Support allows us to take existing Blazor Components such as Blazorise (https://blazorise.com/docs/components/) or a WPF or WinForms application and bundle them into a desktop application using a webview control with access to all underlying hardware APIs. It will allow developers to use web technologies to build desktop applications with access to system resources such as the local file system or a webcam.
+
+### Containerized Images
+
+In .NET 7, the developer's experience with building and deploying containerized applications will be simplified regarding the setup and configuration required to implement secure authentication and authorization while improving the performance of application startup and runtime execution. 
+
+
+![Containerized Images](img/blog/2022-08-23/ContainerizedImages.png align="left")
+
+We also expect to see a preview of Microsoft Orleans, a cross-platform framework for distributed applications. 
+
+## Other Improvements & features
+Let's take a look at some improvements enhancements in .NET 7
+
+### Performance Improvements
+Additional performance improvements in .NET 7 have made it faster and more efficient. Already, .Net 6 has huge improvements in performance with JIT, AOT, inlining method calls, and devirtualization, and .Net 7 will take this even further. This will improve the performance of containerized applications without altering your source code running in the cloud.
+
+### Official HTTP 3 Support
+Http 3 was shipped as a preview feature in .Net 6 & will be a part of .Net 7 & enabled by default. In future .Net 7 preview versions, we’ll see performance improvements & additional TLS features. The HTTP/3 specification isn't finalized and behavioral or performance issues may exist in HTTP/3.
+
+
+![HTTP 3](img/blog/2022-08-23/Http3.png align="left")
+
+### Minimal APIs Improvements
+Minimal APIs were introduced in .Net 6 to allow you to create lightweight APIs without the overhead of controllers. These APIs have seen massive improvements with the introduction of the new Microsoft.AspNetCore.OpenApi package to provide APIs for interacting with the OpenAPI specification in minimal APIs. 
+
+Also, endpoint filters will allow you to implement crosscutting concerns that you can only do with controllers using action filters today.
+
+## Should You Migrate To .NET 7?
```

Comment:
Mention that we will in Blazorise 1.1 support .NET7 and in time we will slowly add features like `required` keyword, INumber interfaces for generic components like NumericEdit and NumericPicker, etc.

For now we will only do the NET7 support.